### PR TITLE
get item types from /get-types endpoint from airtable and handle data FE related #209

### PR DESCRIPTION
**Pull request by Hadeel-Salamin**
- [x] remove mock data and replace it with response from /get-types EP.

- [x] add a removeDuplicate method to remove the value that occurs twice in the drop down.

- [x] apply removeDuplicate on both brands and itemTypes.

- [x] add type id and type name to checkChanges method.

### CODE REVIEW

**Review by FidaaHersh — APPROVED** [review]

**Review by amanshawar — APPROVED** [review]

File: client/src/components/ItemDetails/index.js [hunk 1]
```diff
@@ -241,6 +261,17 @@ class ItemDetails extends Component {
     history.push('/item-list');
   };
 
+  removeDuplicate = (array, item) => {
+    const filteredArray = array;
```
Comment (jema28) [inline client/src/components/ItemDetails/index.js]: why is this needed?

**Review by jema28 — COMMENTED** [review]
See below

File: client/src/components/ItemDetails/index.js (hunk 1 above)
Comment (Hadeel-Salamin) [reply client/src/components/ItemDetails/index.js]: removed and returned with array only

**Review by jema28 — APPROVED** [review]
